microsoft-edge/devtools-guide-chromium/whats-new/2020/06/devtools.md

```diff
@@ -142,7 +142,7 @@ The following sections announce additional features available in Microsoft Edge
 
 The **Styles** pane now has better support for editing styles that were created with the [CSS Object Model (CSSOM)](https://drafts.csswg.org/cssom) APIs.  Many CSS-in-JS frameworks and libraries use the CSSOM APIs under the hood to construct styles.
 
-You can now edit styles added in JavaScript using [Constructable Stylesheets](https://wicg.github.io/construct-stylesheets/).  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
+You can now edit styles added in JavaScript using [Constructable Stylesheets](https://web.dev/constructable-stylesheets/).<!-- todo: https://wicg.github.io/construct-stylesheets/ is 404 -->  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```

Comment:
```suggestion
You can now edit styles added in JavaScript using [Constructable Stylesheets](https://web.dev/constructable-stylesheets/).<!-- todo: https://wicg.github.io/construct-stylesheets/ is 404 -->  Constructable Stylesheets are a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```
@captainbrosset  - the problem with changing the 404 link to https://web.dev/constructable-stylesheets/ is that that page itself contains the 404 link

Reply:
```suggestion
You can now edit styles that were added in JavaScript using [the `CSSStyleSheet` interface](https://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface), which is a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).
```

Reply:
You can now edit styles that were added in JavaScript using the `CSSStyleSheet` interface, which is a new way to create and distribute reusable styles when using [Shadow DOM](https://developer.mozilla.org/docs/Web/Web_Components/Using_shadow_DOM).  See [The `CSSStyleSheet` Interface](https://www.w3.org/TR/cssom-1/#the-cssstylesheet-interface) in _CSS Object Model (CSSOM)_.